resources/js/components/QueryBuilder/QueryCondition.vue

```diff
@@ -100,6 +100,35 @@ export default {
             return this.fields.find(field => field.value === this.condition.attribute);
         },
 
+        getOperatorsForType(fieldValue) {
+            const field = this.getCurrentField();
+
+            if (field && field.operators) {
+                return field.operators.map(operator => ({
+                    value: operator,
+                    label: this.getOperatorLabel(operator)
+                }));
+            }
+
+            if (field) {
+                return this.operators[field.type] || this.operators.text;
+            }
+
+            return this.operators.text;
```

Comment:
```suggestion
            return this.operators[field.type] || this.operators.text;
```

I'm not sure if it needs to be `||` or `??` .